docs/core/tools/dotnet-list-package.md

```diff
@@ -116,7 +116,7 @@ The project or solution file to operate on. If not specified, the command search
 
 - **`--vulnerable`**
 
-  Lists packages that have known vulnerabilities. Cannot be combined with `--deprecated` or `--outdated` options.
+  Lists packages that have known vulnerabilities. Cannot be combined with `--deprecated` or `--outdated` options. Nuget.org is the source of information about vulnerabilities. For more information, see [Vulnerabilities](/nuget/api/registration-base-url-resource) and [How to Scan NuGet Packages for Security Vulnerabilities](/nuget/api/registration-base-url-resource).
```

Comment:
Did you mean to add a different link in the 2nd part?

Reply:
Yes, thanks.